GoogleForCreators/web-stories-wp
# Dashboard: clean up empty text components

## Summary

Consolidating empty stories and no search results views that were done with initial view creation. 

## Relevant Technical Choices
- Now the gutter is set once on the page for content regardless of what is in it. 
- Shared component to handle top margin for text and message styling.

## To-do



## User-facing changes
None, just sharing components instead of repeating them.

## Testing Instructions

See that default no story/template text is intact and when you search for a story that doesn't exist it shows the no search results text. 

---



(Follow up cleanup for design updates from Friday)
Address #4565
